package.json

```diff
@@ -83,38 +83,35 @@
     "underscore.string": "^3.3.4"
   },
   "devDependencies": {
-    "babel-core": "^6.26.0",
+    "@babel/core": "^7.2.2",
+    "@babel/plugin-proposal-class-properties": "^7.3.0",
+    "@babel/plugin-transform-object-assign": "^7.2.0",
```

Comment:
IE support?

Reply:
Ah, actually I didn't check on IE, so even IE11 might not work since preset-env transforms ES2015+ by default.
It looks we can use the specific target polyfill https://babeljs.io/docs/en/babel-preset-env#targets. Personally, I don't use IE, so I didn't really pay attention.

Reply:
@koooge Can you please test this configuration as carefully as possible? A lot of Redash users still use IE10/11/Edge, so it's crucial to keep everything working.

Reply:
We should definitely target Edge and maybe IE11 but not beyond that.

Reply:
@arikfr 

![image](https://user-images.githubusercontent.com/12139186/52492440-de585180-2bd1-11e9-89b1-fcb4dde14579.png)

(IE7 there probably is IE11/Edge in quirks mode)

Reply:
okay I just tested it on win10 ie11 and edge, works fine.